apps/web/src/app/recovery/reset-password/request/page.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import { Button } from '@/src/components/shadcn-ui/button';
+import { Input } from '@/src/components/shadcn-ui/input';
+import { useRouter } from 'next/navigation';
+import { platform } from '@/src/lib/trpc';
+import { useState } from 'react';
+import { toast } from 'sonner';
+
+export default function RequestResetPage() {
+  const [username, setUsername] = useState('');
+  const [email, setEmail] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const router = useRouter();
+
+  const { mutateAsync: sendRecoveryEmail } =
+    platform.account.security.sendRecoveryEmail.useMutation();
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setIsLoading(true);
+    try {
+      await sendRecoveryEmail({ username, email });
+      toast.success(
+        'Recovery email sent. Please check your inbox for the verification code.'
+      );
+      router.push('/recovery/reset-password/verify');
+    } catch (error) {
+      toast.error('An error occurred. Please try again.');
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <div className="flex min-h-screen flex-col items-center justify-center p-4">
+      <h1 className="mb-4 text-2xl font-bold">Request Password Reset</h1>
+      <form
+        onSubmit={handleSubmit}
+        className="w-full max-w-md space-y-4">
```

Comment:
There is no reason to use a form anyways, but if you want to do use the form hook to keep stuff consistent and we can also have the benefits of validation